.github/workflows/pr-create-release-build.yml

```diff
@@ -122,92 +121,92 @@ jobs:
         if: ${{ needs.wait-checks.result == 'success' }}
         run: |
           gh pr comment ${{ needs.create-pr.outputs.pr_number }} --body "/lgtm"
-          gh pr edit ${{ needs.create-pr.outputs.pr_number }} --add-label lgtm
         env:
-         GH_TOKEN: ${{ github.token }}
+          GH_TOKEN: ${{ github.token }}
 
   wait-lgtm:
     runs-on: ubuntu-latest
-    needs: [ check-prev-tag,create-pr,wait-checks,comment-lgtm ]
+    needs: [check-prev-tag, create-pr, wait-checks, comment-lgtm]
     outputs:
       has_lgtm: ${{ steps.wait-lgtm-label.outputs.has_lgtm }} 
+      has_lgtm: ${{ steps.wait-lgtm-label.outputs.has_lgtm }} 
 
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+      has_lgtm: ${{ steps.wait-lgtm-label.outputs.has_lgtm }}
 
-      - name: Wait for lgtm label
+    steps:
+      - name: Wait for LGTM label
+        uses: nick-fields/retry@v3
         id: wait-lgtm-label
-        run: |
-          for i in {1..60}; do
+        with:
+          timeout_minutes: 120
+          max_attempts: 60
+          retry_wait_seconds: 120
+          shell: bash
+          command: |
             LABEL=$(gh pr view ${{ needs.create-pr.outputs.pr_number }} --json labels --jq '.labels[].name')
             echo "Labels: $LABEL"
             if echo "$LABEL" | grep -qw "lgtm"; then
-                has_lgtm=true
-                echo "has_lgtm=${has_lgtm}" >> $GITHUB_OUTPUT
-                break
+              has_lgtm=true
+              echo "has_lgtm=${has_lgtm}" >> $GITHUB_OUTPUT

```

Comment:
where else is this used?

Reply:
"has_lgtm" variable isn't currently utilized in the workflow. During testing it on local, I experimented with incorporating it for the merge part of the code.I  can remove it.